events.js

```diff
@@ -1,17 +1,30 @@
 const luxon = require('luxon');
 const operatingTimezones = require('./operatingTimezones').timezones();
 const isEnabled = require('./isEnabled').isEnabled;
+const turnOffWeekends = require('./turnOffWeekends').turnOffWeekends;
 
 const START_HOUR = parseInt(process.env.HAMMERTIME_START_HOUR || '6', 10);
 const STOP_HOUR = parseInt(process.env.HAMMERTIME_STOP_HOUR || '19', 10);
 
-function getCronHour(hour, zone) {
-  return luxon.DateTime.fromObject({ hour, zone }).setZone('UTC').hour;
+function getCronDayHour(day = '0', hour, zone) {
```

Comment:
It'd be great to have test coverage of this one given it's fairly crucial to scheduling.